{"name": "test_login[admin-]", "status": "failed", "statusDetails": {"message": "AssertionError: 失败原因：密码为空\nassert False == True", "trace": "user = 'admin', psw = ''\n\n    @pytest.mark.parametrize(\"user, psw\", test_login_data)\n    def test_login(user, psw):\n        '''登录用例'''\n        result = login(user, psw)\n>       assert result == True, \"失败原因：密码为空\"\nE       AssertionError: 失败原因：密码为空\nE       assert False == True\n\ntest_demo12.py:23: AssertionError"}, "description": "登录用例", "parameters": [{"name": "user", "value": "'admin'"}, {"name": "psw", "value": "''"}], "start": 1615054278808, "stop": 1615054278808, "uuid": "4eb29ce9-d85c-4967-bec6-e277178b2bd4", "historyId": "bb306a9d57e7e6327bba9c576577dda0", "fullName": "test_demo12#test_login[admin-]", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_demo12"}, {"name": "subSuite"}, {"name": "host", "value": "LHL"}, {"name": "thread", "value": "22228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_demo12"}]}